src/moaiext-tvos/moaiext-tvos.h

```diff
@@ -0,0 +1,18 @@
+//
+//  moaiext-tvos.h
```

Comment:
I'm not sure whether these categories are actually needed.  Might be able to get away without them, but I haven't spent the time to test them.  :cat: